http/jaxrs-reactive/src/main/java/io/quarkus/ts/http/jaxrs/reactive/MediaTypeResource.java

```diff
@@ -0,0 +1,104 @@
+package io.quarkus.ts.http.jaxrs.reactive;
+
+import static io.quarkus.ts.http.jaxrs.reactive.MediaTypeResource.MEDIA_TYPE_PATH;
+import static javax.ws.rs.core.MediaType.APPLICATION_FORM_URLENCODED;
+import static javax.ws.rs.core.MediaType.APPLICATION_SVG_XML;
```

Comment:
Please add static imports for the rest of used `MediaType.` to be in sync with the approach used for the test class.

Alternatively you can drop static imports and use `MediaType.*`, the idea is to have consistent approach in both java files.